docker-compose-3-persona.yml

```diff
@@ -188,16 +194,19 @@ services:
       - energy-owner-node
       - energy-owner-postgres-identity
     environment:
-      - PORT=9010
-      - API_HOST=energy-owner-node
-      - API_PORT=9944
-      - DB_HOST=energy-owner-postgres-identity
-      - DB_PORT=5432
-      - DB_NAME=dscp-identity
-      - DB_USERNAME=postgres
-      - DB_PASSWORD=postgres
-      - SELF_ADDRESS=5FHneW46xGXgs5mUiveU4sbTyGBzmstUspZC92UhjJM694ty
-      - AUTH_TYPE=${AUTH_TYPE:-NONE}
+      - 'PORT=9010'
+      - 'API_HOST=energy-owner-node'
+      - 'API_PORT=9944'
+      - 'DB_HOST=energy-owner-postgres-identity'
+      - 'DB_PORT=5432'
+      - 'DB_NAME=dscp-identity'
+      - 'DB_USERNAME=postgres'
+      - 'DB_PASSWORD=postgres'
+      - 'SELF_ADDRESS=5FHneW46xGXgs5mUiveU4sbTyGBzmstUspZC92UhjJM694ty'
+      - 'AUTH_TYPE=${AUTH_TYPE:-NONE}'
+      - 'API_SWAGGER_BG_COLOR=#e7f6e6'
+      - 'API_SWAGGER_TITLE=IdentityAPIemma'
+      - 'API_SWAGGER_HEADING=Identity-API-System-emma'
```

Comment:
Seems like a funny naming/casing convension. Is there a reason we didn't go with spaces in the end other than you don't like the look?

Also I think we should get the word `HyProof` in there somewhere, so maybe something like `Hyproof Indentity API - Emma`